qiskit_addon_sqd/configuration_recovery.py

```diff
@@ -51,7 +50,7 @@ def post_select_by_hamming_weight(
 
 def recover_configurations(
     bitstring_matrix: np.ndarray,
-    probabilities: Sequence[float],
+    probabilities: np.ndarray,
```

Comment:
I'm ok with restricting this, but I do think any general sequence works here

Reply:
I made this change just to satisfy mypy.

Reply:
Perhaps it is worth accepting either in the type signature? Note that ndarray itself is not a Sequence.